editoast/src/main.rs

```diff
@@ -717,6 +747,92 @@ mod tests {
         tmp_file
     }
 
+    #[actix_test]
+    async fn clone_infra_returns_new_cloned_infra() {
+        // GIVEN
+        let pg_config = PostgresConfig::default();
+        let pg_config_url = pg_config.url().expect("cannot get postgres config url");
+        let manager = ConnectionManager::<PgConnection>::new(pg_config_url.as_str());
+        let db_pool = crate::Data::new(crate::Pool::builder(manager).build().unwrap());
+
+        let infra = Infra {
```

Comment:
Could you use the `small_infra` fixture for those new tests ? This will add cleanup for `created_infra`
You'll still have to manually cleanup the cloned infra